tests/zquantum/qaoa/estimators/cvar_estimator_test.py

```diff
@@ -36,7 +38,11 @@ def custom_run_circuit_and_measure(circuit, n_samples):
             bitstrings = [("0"), ("1")]
             return Measurements(bitstrings)
 
+        def custom_get_wavefunction(circuit):
```

Comment:
I would just replace this whole fixture by:

```
def backend(self):
    return MockQuantumBackend()
```

I don't think we need any of this custom code anymore.

Reply:
I got rid of `custom_run_circuit_and_measure` but I think we still need `custom_get_wavefunction` (it's not implemented in MockQuantumBackend)

Reply:
True, the way it's set up it won't work.
Then perhaps we need to use `SymbolicSimulator` instead of `MockQuantumBackend` 🤷 
It's also in core.